### ISSUE TITLE
Updates of Content and Pre-Commit Hooks

### ISSUE THREAD

**Issue by nesterone** [nesterone/sf-eng-init-gl-spring-2016#452]
[Pull Request](https://github.com/DeuceX/sf-eng-init-gl-spring-2016/pull/4)
1. Run `npm install`
2. Run `npm run lint`
3. Fix all issues with code
4. Commit

After that poin, you won't be able to commit unlinted code

**Comment by DeuceX** [comment]
Thanks, @nesterone  . But it doesn't work for me :(

I merged pull request, and pull'd it from GitHub.  When I tried `npm install` I got some warnings and install didn't work. I tried to delete "node_modules" folder and after this that warnings went away.

Then I tried to write `npm install`, but I got some kind of error ("UNMET PEER DEPENDENCY eslint-plugin -react@5.0.1 extraneous".

I tried to search for this error in the internet, but I didn't find anything useful.
So I updated my node.js to the newest version and I also installed eslint-plugin-react via cmd and after this `npm install` worked:

![img](http://content.screencast.com/users/BragaDmitriy/folders/Jing/media/a6fcc86b-967f-4a9f-8f12-34ac5a2199a1/2016-04-18_1620.png)

After this I wrote not valid code, checked it with the help of `npm run lint`, it said that there are some errors and I tried to commit this file, and commit worked even with lint problems:

![img](http://content.screencast.com/users/BragaDmitriy/folders/Jing/media/17261828-945e-40dd-b769-636a1202e460/2016-04-18_1621.png)
